src/kernels/composable_kernel/src/kernel_wrapper/gridwise_convolution_implicit_gemm_v4r4_gen_xdlops_wrw_fp32_nchw_kcyx_nkhw_lds_double_buffer.cpp

```diff
@@ -0,0 +1,138 @@
+#include "common_header.hpp"
```

Comment:
By creating this new file, we have caused more code duplication. There was need to create a separate wrw solver as it has reduced search space and slightly different tuning parameters. But this file contents' could have remained in the old file